tests/services/test_file_freezer.py

```
@@ -232,3 +234,59 @@ def test_freeze_file_encrypted_success(self, db_session, tmp_path, file_inventor
+        db_session.add(storage_location)
+        db_session.commit()
+
+        inv = file_inventory_factory(path=str(hot_file), storage_type=StorageType.HOT)
+        monitored_path = db_session.get(MonitoredPath, inv.path_id)
+        monitored_path.source_path = str(hot_dir)
+        monitored_path.operation_type = OperationType.MOVE
+
+        captured = {}
+
+        class DummyBackend:
+            def backend_name(self):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `backend_name`

Add return type annotation: `str`

(ANN202)